presto-product-tests/conf/presto/etc/environment-specific-catalogs/singlenode-kerberos-hdfs-impersonation/hive.properties

```diff
@@ -12,13 +12,13 @@ hive.metastore-cache-ttl=0s
 
 hive.metastore.authentication.type=KERBEROS
 hive.metastore.service.principal=hive/hadoop-master@LABS.TERADATA.COM
-hive.metastore.client.principal=hive/hadoop-master@LABS.TERADATA.COM
-hive.metastore.client.keytab=/etc/hive/conf/hive.keytab
+hive.metastore.client.principal=hive/_HOST@LABS.TERADATA.COM
+hive.metastore.client.keytab=/etc/presto/conf/presto-hive.keytab
 
 hive.hdfs.authentication.type=KERBEROS
 hive.hdfs.impersonation.enabled=true
-hive.hdfs.presto.principal=hdfs/hadoop-master@LABS.TERADATA.COM
-hive.hdfs.presto.keytab=/etc/hadoop/conf/hdfs.keytab
+hive.hdfs.presto.principal=presto-server/_HOST@LABS.TERADATA.COM
```

Comment:
please use presto-client/_HOST, we shouldn't use the Presto service principal as the principal to connect to hdfs